EIPS/eip-5489.md

```diff
@@ -0,0 +1,160 @@
+---
+eip: 5489
+title: NFT hyperlink extension
+description: hyperlink extension for EIP-721
+author: IronMan_CH (@coderfengyun)
+discussions-to: https://ethereum-magicians.org/t/eip-5489-nft-hyperlink-extension/10431
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-16
+requires: 165, 721
+---
+
+## Abstract
+
+This EIP proposes a new extension for NFTs (non-fungible token, aka [EIP-721](./eip-721.md)): nft-hyperlink-extention (hNFT), embedding NFTs with hyperlinks, referred to as “hNFTs”. As owners of hNFTs, users may authorize a URL slot to a specific address which can be either an externally-owned account (EOA) or a contract address and hNFT owners are entitled to revoke that authorization at any time. The address which has slot authorization can manage the URI of that slot.
+
+## Motivation
+NFTs have entered mainstream consciousness in a big way as NFT trading volume spiked 21,000% to more than $17 billion in 2021. Despite NFTs’ monumental rise as blockchain-based digital assets, we are just starting to scratch the surface of what is possible with NFTs.
+
+The emergence of NFTs brings more possibilities for creators and Web3 projects to build up decentralized communities, which also shifts the previous company-consumer relationship to more of a partnership. Each NFT holder is entitled to trade, promote, or create based on their NFTs to make profits. 
+
+However, there hasn’t been an effective approach to providing avenues of continued monetization for community holders. Going through all the NFT-related EIPs, the solutions to "making NFTs profitable" are limited to transfers ([EIP-721](./eip-721.md)), rental and mortgage ([EIP-4907](./eip-4907.md)).
+
+Inspired by hyperlink, the essence of World Wide Web, the proposed hNFT standard positions NFT as the medium for Web 3.0. The hNFT standard endows NFTs with a hyperlink extension so that it is possible to exercise the `fructus` right, meanwhile, keeps the `abusus`. With hyperlink extension, NFT owners can earn profit by authorizing any hyperlink slot to an address.
```

Comment:
```suggestion
```
Required: Not motivation.

Reply:
@Pandapip1 How about just keeping "With hyperlink extension, NFT owners can earn profit by authorizing any hyperlink slot to an address." this sentence ?

Reply:
Again, that describes the solution, not the problem, and so would belong in the Abstract.

Reply:
I see, the core problems are:
1. no on-chain way to take advantage of NFT's influence without affecting ownership for now.
2. no standard on how to use in URI in NFT relative standard

Reply:
@Pandapip1 How about just list these two as the motivation?

Reply:
@Pandapip1 I've modified the motivation.